content/docs/0.6.0/reference/monitoring/dynatrace/index.md

```diff
@@ -168,6 +168,54 @@ This means that Dynatrace will automatically apply tags to your onboarded servic
 
 In addition, a *Problem Notification* has automatically been set up to inform Keptn of any problems with your services to allow auto-remediation. This will be described in more detail in the [Runbook Automation](../../usecases/runbook-automation-and-self-healing/) tutorial. You can check the problem notification by navigating to **Settings > Integration > Problem notifications** and you will find a **keptn remediation** problem notification.
 
+## Setup Dynatrace SLI provider
+
+During the evaluation of a quality gate, the Dynatrace SLI provider is required that is implemented by an internal Keptn service, the *dynatrace-sli-service*. This servcie will fetch the values for the SLIs that are referenced in a SLO configuration.
```

Comment:
Same as with prometheus, can you please add a quick check here if Dynatrace-sli-service is already installed:

```console
kubectl get deployment -n keptn -o wide dynatrace-sli-service
```

Reply:
Good idea. I added this step.